apiserver/plane/api/views/project.py

```
@@ -250,7 +250,7 @@
                 serializer.save()
                 self.create_project(
                     request.META.get("HTTP_ORIGIN"), 
-                    user, 
+                    request.user, 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)